### DISCUSSION TITLE
Reset database (mockgoose.helper.reset(); equivalent)

### DISCUSSION THREAD

**Discussion by ismail-codinglab** [typegoose/mongodb-memory-server#452] [Q&A] [answered]
## What is your question?
How do i reset the database (a.k.a. empty all of the collections)

**Comment by hasezoey** [comment, marked as answer]
here what can be done, in order (from 1 best to lowest) that should be done
1. remove the database with the mongodb driver (or mongoose or anything else) [this should be preferred]
2. use a different database in the connection string (like generate an name with something like `uuidv4`)
3. restart the instance (only work if storageEngine is `ephermeralForTest`, for others the files need to be also cleaned) [this should be avoided, because it would slow down test]
